e2e/login.spec.ts

```
@@ -43,3 +46,28 @@ for (const project of projects) {
   });
 }
+
+for (const project of projects) {
+  test(`As User with pro rights, I can log in to ${project.title}, check that this profile is mine, and log out`, async ({
+    pageManager,
+  }) => {
+    await pageManager.atBrowser.openProject(project);
+    await pageManager.atNavigationMenu.checkThereIsNoTextInNavigationMenu('Profile');
+    await pageManager.atNavigationMenu.clickButtonToOpenPage('Login');
+    await pageManager.atLoginPage.typeLoginPasswordAndLogin(
+      process.env.EMAIL_PRO!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)